src/nbl/asset/interchange/CGLTFLoader.cpp

```diff
@@ -1453,8 +1454,92 @@ namespace nbl
 							pushConstants.availableTextures |= CGLTFPipelineMetadata::SGLTFMaterialParameters::EGT_NORMAL_TEXTURE;
 
 							auto cpuNormalTexture = IMAGE_VIEWS[SGLTF::SGLTFMaterial::EGT_NORMAL_TEXTURE];
+							{
```

Comment:
@Crisspl review the changes from this commit

Reply:
oh shit, missed this comment